cct/lib/git.py

```diff
@@ -8,19 +9,25 @@
 
 
 def clone_repo(url, path, version=None, force=False):
+    # this is not a nice way, but we are passing version as a None explictly on multiple places
+    if not version:
+        version = "master"
     try:
-        if not os.path.exists(path):
-            logger.info("Cloning %s into %s" % (url, path))
-            subprocess.check_call(["git", "clone", url, path])
-            if version:
-                logger.info('Checking out %s revision' % version)
-                subprocess.check_call(['git', 'checkout', version], cwd=path)
-        elif os.path.exists(path) and force:
-            logger.info('Forcing %s revision for %s' % (version, path))
-            subprocess.check_call(['git', 'checkout', version], cwd=path)
+        if os.path.exists(path):
+            if force:
+                logger.info("Removing old module from path: '%' - forced version will be fetched." % path)
```

Comment:
This log message is a bit unnecessary. I would just leave the first part of the sentence.